docs/streams.md

```diff
@@ -98,105 +90,94 @@ example, if you had the following events in a stream :
 3@test-stream
 ```
 
-If you set the `TruncateBefore` or `$tb` value to 3, a read of the stream would result in only reading the
-last event:
+Setting the **`TruncateBefore`** (or **`$tb`**) value to 3, results in reading only the last event: 
 
 ```
 3@test-stream
 ```
 
-A **soft delete** makes use of `TruncateBefore` and `$tb`. When you delete a stream, its `TruncateBefore`
-or `$tb` is set to
-the [max long/Int64 value](https://docs.microsoft.com/en-us/dotnet/api/system.int64.maxvalue?view=net-5.0):
-9223372036854775807. When you read a soft deleted stream, the read returns a `StreamNotFound` or `404` result.
-After deleting the stream, you are able to append to it again, continuing from where it left off.
+A **soft delete** leverages **`TruncateBefore`** (or **`$tb`**). When you delete a stream, **`TruncateBefore`**
+(or **`$tb`**) is set to the [max long/Int64 value](https://docs.microsoft.com/en-us/dotnet/api/system.int64.maxvalue?view=net-5.0): 9223372036854775807. 
+
+Appending to the stream adds events starting from event number 4, the last stream revision before deletion, incremented by one.  
+
+When you read a soft-deleted stream, the read returns a **`StreamNotFound`** or **`404`** result. After deleting the stream, you can append to it again, continuing from where it left off.
 
-For example, if you soft deleted the above example stream, the `TruncateBefore` or `$tb` is set to
-9223372036854775807. If you were to append to the stream again, the next event is appended with event number
-4. Only events from event number 4 (last stream revision before deleting, incremented by one) onwards are
-visible when you read this stream.
 
 ### Max count and Max age
 
-**Max count** (`$maxCount` and `MaxCount`) limits the number of events that you can read from a stream. If you
-try to read a stream that has a max count of 5, you are only able to read the last 5 events, regardless of how
-many events are in the stream.
+**Max count** (**`$maxCount`** and **`MaxCount`**) restricts the number of events you can read from a stream. Reading a stream with a max count of 5 allows you to retrieve only the last 5 events, regardless of the total events in the stream. 
 
-**Max age** (`$maxAge` and `MaxAge`) specifies the number of seconds an event can live for. The age is
-calculated at the time of the read. So if you read a stream with a `MaxAge` of 3 minutes and one of the events
-in the stream has existed for 4 minutes at the time of the read, it is not returned.
+**Max age** (**`$maxAge`** and **`MaxAge`**) specifies the time an event can live in seconds. The age is
+calculated at the time of the read. For example, if you read a stream with a **`MaxAge`** of 3 minutes, an event existing for 4 minutes won't be returned.
 
 ### Hard delete
```

Comment:
why do we have hard delete defined in 2 separate places? Should we take the details out of line 75 and just mention that a hard deleted stream cannot be reopened up there?

Reply:
Honestly, I'd like to change the whole thing and I'm sure we will when we restructure things - but for now, I've removed it from line 75 and added a phrase to the hard delete section.